### PR TITLE
1st-commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
1rst-commit

### CODE REVIEW

**Comment by pavel-gutsal** [comment]
working form (sort of)
https://pavel-gutsal.github.io/layout_html-form/
https://pavel-gutsalt.github.io/layout_html-form/report/html_report/

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,235 @@
...
+            name="love cats"
+            value="yes"
+            id="catLover"
+          >
+          Yes
+          <input
+            class="radio"
+            type="radio"
+            name="loveCats"
+            value="no"
```
Comment (aliceblunt8) [inline src/index.html:141]: name should be the same

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]
only one option can be chosen at a time, also by clicking on the label yes or no the corresponding option should be marked

Screenshot 2022-06-02 at 17 52 07

File: src/index.html (hunk 1 above)
Comment (pavel-gutsal) [reply src/index.html]: fixed

**Comment by pavel-gutsal** [comment]
https://pavel-gutsal.github.io/layout_html-form/

**Comment by pavel-gutsal** [comment]
TEST REPORT LINK is operational . I made a typo initially and didn`t noticed it.

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**